src/einsteinpy/tests/test_utils/test_christoffel_wo_sympy.py

```diff
@@ -0,0 +1,11 @@
+import os
+import sys
+
+import pytest
+
+
+def test_christoffel_wo_sympy():
+    sys.modules["sympy"] = None
+    from einsteinpy.utils import christoffel  # isort:skip
```

Comment:
Well, what is the test doing? don't you feel that you should catch the output and compare it with the pre-existing string ?

Reply:
Removed the test